mlflow/tensorflow.py

```diff
@@ -694,12 +694,30 @@ def _setup_callbacks(lst):
+    | ``tf.keras.callbacks.earlyStopping``       | Metrics from the ``EarlyStopping`` callbacks. For example,  | ``fit() or fit_generator()`` parameters associated with ``EarlyStopping``.         | --            | --                                                                                                                                               |
+    |                                            | stopped_epoch, restored_epoch, restore_best_weight, etc     | For example, min_delta, patience, baseline, restore_best_weights,etc               |               |                                                                                                                                                  |
+    +--------------------------------------------+-------------------------------------------------------------+------------------------------------------------------------------------------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | ``tf.estimator``                           | TensorBoard metrics. For example average_loss, loss etc     | steps, max_steps                                                                   | --            | `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (TF saved model) on call to ``tf.estimator.export_saved_model``                     |
+    +--------------------------------------------+-------------------------------------------------------------+------------------------------------------------------------------------------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | TensorFlow Core                            | All ``tf.summary.scalar`` calls                             | --                                                                                 | --            | --                                                                                                                                               |
+    +--------------------------------------------+-------------------------------------------------------------+------------------------------------------------------------------------------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+
+    Note that autologging for ``tf.keras`` is handled by :py:func:`mlflow.tensorflow.autolog`, not :py:func:`mlflow.keras.autolog`.
```

Comment:
Same as before, would move this statement above the table.

Reply:
This bit will have to be integrated with the first sentence. It can't come below "Autologging captures the following information:"; it'll look awkward. 

I suggest the following workaround:
"Enables automatic logging from TensorFlow to MLflow. Note that autologging  .....

Autologging captures the following information:"
This sentence flow is better.